documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -3722,26 +3776,35 @@ The connector's `table.include.list` or `table.exclude.list` configuration optio
 [id="oracle-pga-aggregate-limit"]
 === ORA-04036: PGA memory used by the instance exceeds PGA_AGGREGATE_LIMIT
 
-Oracle may issue this error if you're connecting to a database that changes very infrequent.
+Oracle might report this error when {prodname} connects to a database in which changes occur infrequently.
 The {prodname} connector starts an Oracle LogMiner session and reuses this session until a log switch is detected.
-The reuse is both a performance and resource utilization optimization; however, a long-running mining session can cause high PGA memory usage.
+The reuse is both a performance and resource utilization optimization; however, a long-running mining session can cause high Program Global Area (PGA) memory usage.
 
-If your redo log switch frequency is low, you can avoid the ORA-04036 error by configuring Oracle to automatically switch logs on a defined frequency.
-A log switch causes the connector to restart the mining session, therefore avoiding high PGA memory usage.
-The following configuration forces Oracle to switch log files every 20 minutes if a log switch hasn't occurred within that window:
+If your redo log switches infrequently, you can avoid the ORA-04036 error by specifying how frequently Oracle switches logs.
+A log switch causes the connector to restart the mining session, thereby avoiding high PGA memory usage.
+The following configuration forces Oracle to switch log files every 20 minutes if a log switch does not occur during that interval:
 
 [source,sql]
 ----
 ALTER SYSTEM SET archive_lag_target=1200 scope=both;
 ----
 
-This change will likely require SYSDBA permissions, so coordinate with your database administrator.
+Running the preceding query specific administrative privileges.
+Coordinate with your database administrator to implement the change.
+
+<<<<<<< Updated upstream
+=======
+As alternative to adjusting the Oracle `ARCHIVE_LAG_TARGET` parameter, you can limit the duration of an Oracle LogMiner session by setting the connector configuration option xref:oracle-property-log-mining-session-max-ms[`log.mining.session.max.ms`].
+The `log.mining.session.max.ms` option causes the LogMiner session to restart regularly, whether or not the switch to a new database log occurs.
 
+
+>>>>>>> Stashed changes
```

Comment:
This seems strange to have these `<<<<< Updated upstream` and `>>>>> Stashed changes` lines.  If these are being pulled back from upstream into 1.9, it would make more sense to backport the original commits from main to 1.9 if that's the case.

Reply:
Oh, shoot. I didn't realize those were there. Ok, fix upcoming. Thanks.

Reply:
Those stash indicators are artifacts of the problems I was having last night when I tried to switch the base branch for my changes. 

That content that was added post-1.9, so I assume it only applies to the subsequent releases, right @Naros?

Reply:
No, DBZ-4963 introduced the `log.mining.session.max.ms` setting and was only applied to `main` and not backported to the 1.9 branch, so this cannot exist in the 1.9 branch at all.

Reply:
ok, got it. Thanks.